packages/editor/src/components/inputs/InteractableGroup.tsx

```diff
@@ -44,162 +49,127 @@ const InteractableTheme = [
-    case 'infoBox':
-      return (
-        <>
-          <InputGroup name="Interaction Theme" label={t('editor:properties.interaction.theme')}>
-            <SelectInput
-              options={InteractableTheme}
-              value={node.interactionThemeIndex}
-              onChange={onChangeInteractionThemeIndex}
-            />
-          </InputGroup>
-          <InputGroup name="Interaction Name" label={t('editor:properties.interaction.name')}>
-            <StringInput value={node.interactionName} onChange={onChangeInteractionName} />
-          </InputGroup>
-          <InputGroup name="Interaction Description" label={t('editor:properties.interaction.description')}>
-            <StringInput value={node.interactionDescription} onChange={onChangeInteractionDescription} />
-          </InputGroup>
-          <ArrayInputGroup
-            name="Interaction Images"
-            prefix="Image"
-            values={node.interactionImages}
-            onChange={onChangeInteractionImages}
-            label={t('editor:properties.interaction.images')}
-            acceptFileTypes={ImageFileTypes}
-            itemType={ItemTypes.Images}
-          ></ArrayInputGroup>
-          <ArrayInputGroup
-            name="Interaction Videos"
-            prefix="Video"
-            values={node.interactionVideos}
-            onChange={onChangeInteractionVideos}
-            label={t('editor:properties.interaction.videos')}
-            acceptFileTypes={VideoFileTypes}
-            itemType={ItemTypes.Videos}
-          ></ArrayInputGroup>
-          <ArrayInputGroup
-            name="Interaction Urls"
-            prefix="Url"
-            values={node.interactionUrls}
-            onChange={onChangeInteractionUrls}
-            label={t('editor:properties.interaction.urls')}
-          ></ArrayInputGroup>
-          <ArrayInputGroup
-            name="Interaction Models"
-            prefix="Model"
-            values={node.interactionModels}
-            onChange={onChangeInteractionModels}
-            label={t('editor:properties.interaction.models')}
-            acceptFileTypes={ModelFileTypes}
-            itemType={ItemTypes.Models}
-          ></ArrayInputGroup>
-        </>
-      )
-    default:
-      break
+  // function to handle changes in payloadHtmlContent
+  // const onChangePayloadHtmlContent = (payloadHtmlContent) => {
+  //   const sanitizedHTML = dompurify.sanitize(payloadHtmlContent)
+  //   if (sanitizedHTML !== payloadHtmlContent)
+  //     console.warn("Code has been sanitized, don't try anything sneaky please...")
+  //   CommandManager.instance.setPropertyOnSelectionEntities({
+  //     component: InteractableComponent,
+  //     properties: { payloadHtmlContent: sanitizedHTML }
+  //   })
+  // }
+
+  const interactableComponent = getComponent(props.node.entity, InteractableComponent)
+
+  const renderInteractableTypeOptions = () => {
+    switch (interactableComponent.interactionType) {
+      case 'infoBox':
```

Comment:
why do we need a switch statement here if there is only 1 case that we need to handle?